hasher-matcher-actioner/hmalib/common/aws_dataclass.py

```diff
@@ -81,8 +86,14 @@ def py_to_aws(py_field: t.Any, in_type: t.Optional[t.Type[T]] = None) -> T:
     if in_type == bool:  # BOOL
         return py_field  # type: ignore # mypy/issues/10003
     if in_type == t.Set[str]:  # SS
+        if not py_field or len(py_field) == 0:
+            # Handle empty sets.
+            return {EMPTY_SET_SENTINEL_VALUE_STR}  # type:ignore
         return py_field  # type: ignore # mypy/issues/10003
     if in_type == t.Set[int]:  # SN
+        if not py_field or len(py_field) == 0:
```

Comment:
What are the cases where the first statement can be false be the second statement is true? Don't we expected `py_field` to be a collection object?

Reply:
You're correct. The second one should suffice.